### PR TITLE
Job model interface in ts

### PR DESCRIPTION
**Pull request by beining00**
Add the job Model interface to `postProcessHandler` and `uploadHandler`

### CODE REVIEW

**Review by YinghaoMa — APPROVED** [review]

**Review by mt-empty — CHANGES_REQUESTED** [review]
Great job, the job model look good.
I only looked at the diff, I didn't pull and test(frankly I wouldn't know how to test it).

File: server/postProcessHandler/src/service/jobModel.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export interface job {
+	id: string;
+	email: string;
+	uploadTime: string;
+	files: file[];
+}
+
+export interface file {
+	fileReference: string;
+	fileType: FileType;
```
[the remark this answers is not in the document]
Comment (beining00) [reply server/postProcessHandler/src/service/jobModel.ts:10]: enum should have it first letter capitalised I guess

[the remark this answers is not in the document]
Comment (mt-empty) [reply server/postProcessHandler/src/service/jobModel.ts:10]: Ok, I think `fileStatus` should also be capitalized

**Review by teva0003 — APPROVED** [review]

**Review by mt-empty — APPROVED** [review]

File: server/postProcessHandler/src/service/jobModel.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export interface job {
```
Comment (reteppeter) [inline server/postProcessHandler/src/service/jobModel.ts:1]: These interfaces don't have the type information that DynamoDB returns/uses (AttributeValue).

**Pull request closed without merging**